app/lib/actions/katello/capsule_content/create_or_update.rb

```diff
@@ -51,6 +51,14 @@ def repository_relative_path(repository)
             repository.relative_path
           end
         end
+
+        def repository_download_policy(repository)
+          if repository.is_a?(::Katello::Repository)
```

Comment:
@ehelms since CVPE does not have a library instance, I am seeing

```undefined local variable or method `repository' for #<Katello::ContentViewPuppetEnvironment:0x000000054718f0>```